connexion/operation.py

```diff
@@ -92,9 +91,6 @@ def security_decorator(self):
                 logger.debug(
                     "... Security type '%s' not natively supported by Connexion; you should handle it yourself",
                     security_definition['type'], extra=vars(self))
-            else:
-                logger.warning("... Security type '%s' unknown. **IGNORING SECURITY REQUIREMENTS**",
-                               security_definition['type'], extra=vars(self))
```

Comment:
An invalid security definition in the swagger specification will throw now an error while validating against swagger_spec_validator so this code won't be reachable.